### PR TITLE
Feature/html css update

### PR DESCRIPTION
**Pull request by bk7711**

### CODE REVIEW

File: homepage.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <head>
         <meta charset="utf-8">
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
-        <title></title>
+        <title>Let's bring out yout inner chef!</title>
```
Comment (bk7711) [inline homepage.html:8]: new title

File: homepage.html [hunk 2]
```diff
@@ -13,23 +13,30 @@
     </head>
     <body>
        <!-- start of navbar -->
-       <nav class="grey darken-2">
-        <div class="nav-wrapper">
-          <a href="#" class="brand-logo">Logo</a>
+       <div class="navbar-fixed">
+         <nav class="grey darken-2">
+          <div class="nav-wrapper">
```
Comment (bk7711) [inline homepage.html]: update to header

File: homepage.html [hunk 3]
```diff
@@ -13,23 +13,30 @@
...
-      </nav>
       <!-- end of navbar -->
       <header class="header grey">
+        <p class="text-center blue-text text-darken-2">
+          What to make tonight? Let's get some ideas and we'll get recipes and tutorials going
+        </p>
 
       </header>
       <main class="row">
-        <div class="col s12 m3 search"></div>
```
Comment (bk7711) [inline homepage.html]: clean formatting

File: homepage.html [hunk 4]
```diff
@@ -13,23 +13,30 @@
...
-        <div class="col s10 m9 display"></div>
```
Comment (bk7711) [inline homepage.html]: clean formatting

File: homepage.html [hunk 5]
```diff
@@ -38,8 +45,8 @@
         <div class="container">
           <div class="row">
             <div class="col l6 s12">
-              <h5 class="white-text">Footer Content</h5>
-              <p class="grey-text text-lighten-4">You can use rows and columns here to organize your footer content.</p>
+              <h5 class="white-text">Foodie delight</h5>
```
Comment (bk7711) [inline homepage.html:93]: update footer

File: homepage.html [hunk 6]
```diff
@@ -38,8 +45,8 @@
...
+              <p class="grey-text text-lighten-4">get your ideas going in kitchen!</p>
```
Comment (bk7711) [inline homepage.html:94]: great title

File: homepage.html [hunk 7]
```diff
@@ -59,6 +66,6 @@ <h5 class="white-text">Links</h5>
       </footer>
       <!-- end of footer -->
         <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
-        <script src="" async defer></script>
+        <script src="./assets/js/script.js"></script>
```
Comment (bk7711) [inline homepage.html]: updated formatting

File: homepage.html [hunk 8]
```diff
@@ -13,23 +13,30 @@
...
-          <a href="#" class="brand-logo">Logo</a>
+       <div class="navbar-fixed">
+         <nav class="grey darken-2">
+          <div class="nav-wrapper">
+          <a href="#" class="brand-logo">the foodie's delight</a>
           <ul id="nav-mobile" class="right hide-on-med-and-down">
-            <li><a href="sass.html">Home</a></li>
-            <li><a href="badges.html">Component 1</a></li>
-            <li><a href="collapsible.html">Component 2</a></li>
+            <li class="active"><a href="./homepage.html">Home</a></li>
```
Comment (bk7711) [inline homepage.html]: update

**Review by bk7711 — COMMENTED** [review]
These are great. Thanks!

File: homepage.html (hunk 7 above)
Comment (RayWillie) [reply homepage.html]: awesome!

**Review by RayWillie — COMMENTED** [review]
This looks great!

**Pull request merged**